S3/Config.py

```diff
@@ -273,7 +273,8 @@ def __init__(self, configfile = None, access_key=None, secret_key=None, access_t
                 self.secret_key = secret_key
             if access_token:
                 self.access_token = access_token
-                # Do not refresh the IAM role when an access token is provided.
+            # Do not refresh the IAM role when an access token is provided.
+            if self.access_token:
```

Comment:
It's basically equal to previous if statement

Reply:
I don't think that it is the case if i'm not wrong.

Without this fix, if the access_token is set directly in the configuration file (and not set with command line or env var); then the access_token_refresh will not be disabled.

But, I'm wondering if there is not a side effect to this. Something like the goal was especially to not be stuck if an access token was written in the file. Especially for the case of a "configure" that would write the access token.